.github/workflows/ci-smart-contract.yml

```diff
@@ -0,0 +1,181 @@
+      - name: Start local node
+        id: start
+        run: |
+          fnsad start &
+          sleep 1
+      - name: CI test
+        run: bash ./ci_test/queue_ci.sh
+        shell: bash
+      - name: Stop local node
+        if: ${{ always() && steps.start.conclusion == 'success' }}
+        run: kill $(ps -ef | grep "fnsad start" | grep -v grep | awk '{print $2}')
+  
+  build-finschia-docker-image:
+    runs-on: ubuntu-latest
+    needs: check-diffs
+    if: ${{ needs.check-diffs.outputs.get_diff }}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: docker/setup-buildx-action@v2
+      - name: Build and export
+        uses: docker/build-push-action@v4
+        with:
+          file: Dockerfile
+          context: .
+          tags: finschia/finschianode:latest
+          outputs: type=docker,dest=/tmp/finschia-ci-test.tar
+      - name: Upload artifact
+        uses: actions/upload-artifact@v3
+        with:
+          name: finschia-ci-test
+          path: /tmp/finschia-ci-test.tar
+  
+  test-smart-contract-in-docker-container:
+    runs-on: ubuntu-latest
+    needs: [build-finschia-docker-image, get-cosmwasm-releases]
+    strategy:
+      fail-fast: false
+      matrix:
+        versions: ${{ fromJson(needs.get-cosmwasm-releases.outputs.cosmwasm_versions) }}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: docker/setup-buildx-action@v2
+      - name: Download smart contracts
+        uses: dsaltares/fetch-gh-release-asset@master
+        with:
+          repo: 'Finschia/cosmwasm'
+          version: tags/${{ matrix.versions }}
+          regex: true
+          file: '.*'
+          target: 'contracts/'
+      - name: Download docker image
+        uses: actions/download-artifact@v3
+        with:
+          name: finschia-ci-test
+          path: /tmp
+      - name: Load image
+        run: |
+          docker load --input /tmp/finschia-ci-test.tar
+          docker image ls -a
+      - name: Set configure file
+        run: ./init_single.sh docker
+        shell: bash
+      # sleep 5 seconds to wait for finschia to start up
+      - name: Start docker container
+        id: docker_start
+        run: |
+          container_id=$(docker run -d \
+            -p 26656:26656 -p 26657:26657 \
+            -v /home/runner/.finschia:/root/.finschia \
+            -v $(pwd)/ci_test:/root/ci_test \
+            -v $(pwd)/contracts:/root/contracts \
+            finschia/finschianode fnsad start)
+          echo "container_id=$container_id" >> "$GITHUB_OUTPUT"
+          sleep 5
+      - name: CI test
+        run: |
+          docker exec ${{ steps.docker_start.outputs.container_id }} apk add --no-cache jq bash && \
+          docker exec ${{ steps.docker_start.outputs.container_id }} bash ./ci_test/queue_ci.sh
+      - name: Stop docker container
+        if: ${{ always() && steps.docker_start.conclusion == 'success' }}
+        run: docker stop ${{ steps.docker_start.outputs.container_id }}
+        
```

Comment:
Would you just put the LF only in the last line? There are unnecessary spaces in the last line. 

(So I can see the below)
<img width="115" alt="screenshot 2023-07-20 11 10 42" src="https://github.com/Finschia/finschia/assets/1164270/7c1ad41a-14ea-4442-a92e-db63345ad3c2">